src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Assembly.cs

```diff
@@ -563,7 +563,7 @@ class Program
 {
     void M(Test x) {}
 }
-", options: TestOptions.ReleaseDll, references: new[] { MscorlibRef_v4_0_30316_17626, hash_module });
+", options: TestOptions.ReleaseDll, references: new[] { MscorlibRef_v45, hash_module });
```

Comment:
I have seen several cases where binaries of different versions were combined together for example. With 46 it would cause failures because for example ExtensionAttribute has moved between assemblies and you may end up with two of them or none if mix 45 and 46.